kitsune/users/tests/test_migration_0033.py

```diff
@@ -0,0 +1,173 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.db.models import Q
+
+from kitsune.questions.tests import AnswerFactory, QuestionFactory
+from kitsune.questions.models import QuestionVote, AnswerVote
+from kitsune.users.tests import ProfileFactory
+from kitsune.wiki.tests import RevisionFactory, DocumentFactory
+from kitsune.messages.models import InboxMessage, OutboxMessage
+from kitsune.gallery.tests import ImageFactory, VideoFactory
+from kitsune.forums.tests import ThreadFactory, PostFactory
+
+
+class TestDeleteNonMigratedUsersMigration(TestCase):
```

Comment:
I'm thinking that we should change/modify the name to suggest that we're not really testing the migration exactly as it is, but instead we're testing the query used within the migration.